backend/api/v1/endpoints/auth.py

```
@@ -61,15 +53,25 @@ async def login(form_data: OAuth2PasswordRequestForm = Depends(), db: Session =
 
-        return {"username": username}
+        if not username:
+            raise HTTPException(status_code=401, detail="Username not found in token")
+        user = db.query(models.User).filter(models.User.username == username).first()
+        
+        # If user is not found, raise an error
+        if not user:
+            raise HTTPException(status_code=404, detail="User not found")
+
+        # Return the user's id
+        return {"id": user.id, "username": user.username}
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)